MANIFEST.in

```diff
@@ -0,0 +1 @@
+include eplaunch/interface/main_icon.ico
```

Comment:
When building a Python wheel, you need to declare non-python-related files that should be included.  This manifest does just that.